components/tooltip/README.md

```diff
@@ -151,3 +151,44 @@ In the following example to constrain the tooltip to the dashed boundary we can
   </d2l-tooltip>
 </div>
 ```
+
+## Help Tooltip [d2l-tooltip-help]
+
+The `d2l-tooltip-help` component is used to display additional information when users focus or hover over some text formatted with an underline.
+
+<!-- docs: demo code name:d2l-tooltip-help autoSize:false size:small -->
+```html
+<script type="module">
+  import '@brightspace-ui/core/components/tooltip/tooltip-help.js';
+</script>
+
+<d2l-tooltip-help text="Helpful label">Contents should elaborate on the label (be short and concise)</d2l-tooltip-help>
+```
+
+<!-- docs: start hidden content -->
+### Properties
+
+| Property | Type | Description |
+|--|--|--|
+| `text` | String, required | Provide the `text` that will render as the Help Tooltip opener.|
+
+### Events
+* `d2l-tooltip-show`: dispatched when the opener text is hovered over, focused on, or clicked
+* `d2l-tooltip-hide`: dispatched when the opener text is no longer hovered over or focused on
+<!-- docs: end hidden content -->
+
+### Best Practices
+
+<!-- docs: start best practices -->
+<!-- docs: start dos -->
+* Use a helpful label that provides value on its own
+* The contents of the tooltip should elaborate on the label
+* Keep help text short and concise (full sentences are not necessary)
+* Use a help tooltip when there are space limitations, such as in a table, list, or narrow sidebar
+<!-- docs: end dos -->
+
+<!-- docs: start donts -->
+* Don’t overuse help tooltips (users end up hunting for information and even experts feel obligated to check their contents)
+* Don't use help tooltips when you are able to use inline help text instead
+<!-- docs: end donts -->
+<!-- docs: end best practices -->
```

Comment:
Does it make sense to have another Best Practices section for the `Help Tooltip` when we already have one at the top for tooltips in general? (are we strict with having 1 "Best Practices" per component page?)

Do we always need to start off the Dos with `"Use..."` and Don'ts with `"Don't"`? (not currently being done above)

In addition to any other changes to this PR, I'm especially open to suggestions for revising the dos and donts content as well (perhaps removing what's in the brackets, or perhaps rewording some things)

Let me know how good or bad it currently sounds! 🙂

Thank you @geurts for providing [a lot of the content](https://www.figma.com/proto/A8jk5soJf1bz5z560Uld32/Help-Tooltips?page-id=0%3A1&node-id=2%3A100&viewport=469%2C216%2C0.32&scaling=min-zoom&starting-point-node-id=2%3A98) here.